### PR TITLE
[PT-74]-Waitlist

### PR DESCRIPTION
**Pull request by ItsFaizan**
### Description: 
Waitlist Page added having firstname, lastname, email, yearsOfExperience, toolUsage as fields 

[PT-74]: https://0xbase.atlassian.net/browse/PT-74?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by fuddin95 — COMMENTED** [review]
Need to add 
- Form validation on email field
- Add waitlist to Navbar instead of Contact Us 
- Title should be Center Aligned
- Make the form in a container directory `WaitlistForm.tsx`
- use LoadingButton once the user clicks on `join waitlist` button
- Spacing is uneven between Text field and the Button

File: src/container/WaitlistForm.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    console.log(formData)
+
+    if (target.name === 'email' && validateEmail(target.value)) {
+      setEmailError('')
+    }
+  }
+
+  const validateEmail = (email: string) => {
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/
+    return emailRegex.test(email)
```
Comment (fuddin95) [inline src/container/WaitlistForm.tsx:42]: move this functions to `utils/helper.ts` file

File: src/container/WaitlistForm.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+          margin="normal"
+        />
+        <TextField
+          label="Email"
+          name="email"
+          value={formData.email}
+          onChange={handleChange}
+          fullWidth
+          margin="normal"
+          error={!!emailError}
```
Comment (fuddin95) [inline src/container/WaitlistForm.tsx:104]: what does `!!`  do?

Comment (ItsFaizan) [reply src/container/WaitlistForm.tsx:104]: It is other form for writing error={emailError !== ' '},  if it contains some error message, the error prop will be set to true

File: src/container/WaitlistForm.tsx (hunk 1 above)
Comment (ItsFaizan) [reply src/container/WaitlistForm.tsx:42]: Done

**Review by fuddin95 — APPROVED** [review]

**Pull request merged**